extensions/amp-story-page-attachment/0.1/amp-story-page-attachment.js

```diff
@@ -428,9 +432,18 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
     this.toggleCloseButtonTabIndex_(true);
 
     this.analyticsService_.triggerEvent(StoryAnalyticsEvent.OPEN, this.element);
-    this.analyticsService_.triggerEvent(
-      StoryAnalyticsEvent.PAGE_ATTACHMENT_ENTER
-    );
+
+    if (childElementByTag(this.pageEl_, 'amp-story-shopping-attachment')) {
+      this.analyticsService_.triggerEvent(
+        StoryAnalyticsEvent.PAGE_ATTACHMENT_ENTER,
+        null,
+        {'calledFrom': 'story-shopping-shop-now'}
```

Comment:
I wonder if we can distinguish between cta and shopping tag clicks.
This way we could remove the `SHOPPING_TAG` event.

Reply:
Yes, there's a way to distinguish it here:
https://github.com/ampproject/amphtml/pull/37856/commits/86bec37089f659134aeac4bb2230d19460f45520
Let me know if you'd like me to implement this approach and I'll do it.

Reply:
I found a way to do it using `document.activeElement`, have updated with the the new code.